crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/factory/MeansAssessmentServiceFactory.java

```diff
@@ -18,7 +16,10 @@ public AssessmentService getService(AssessmentType assessmentType) {
         if (AssessmentType.INIT.equals(assessmentType)) {
             return applicationContext.getBean(InitMeansAssessmentService.class);
         } else {
-            return applicationContext.getBean(FullMeansAssessmentService.class);
+            EligibilityChecker crownCourtEligibilityService = applicationContext.getBean(CrownCourtEligibilityService.class);
+            AssessmentCriteriaChildWeightingService childWeightingService = applicationContext.getBean(AssessmentCriteriaChildWeightingService.class);
+
+            return new FullMeansAssessmentService(crownCourtEligibilityService, childWeightingService);
```

Comment:
The eligibility checker is now an interface. How is the autowiring going to pick up the right class? Happy to remove, I wasn't confident that this worked properly otherwise - lots of tests use Mocks

Reply:
Ok - I'm really unsure about this auto-wiring stuff, so I went for something I knew worked. Have reverted